docs/syntax.rst

```diff
@@ -164,12 +164,11 @@ One of the most common cases when a localizer needs to use a placeable is when
 there are multiple variants of the string that depend on some external 
 argument.
 
-FTL provides a select expression that chooses one of the provided variants 
-based on the given selector.
+FTL provides the select expression syntax, which chooses one of the provided
+variants based on the given selector.
 
-By default, when a number is used as a selector, FTL implicitly uses ``PLURAL`` 
-formatter that selects the proper plural case for a given language. In English 
-it will be either ``one`` or ``other``.
+When the selector is a number FTL implicitly converts it into the proper plural 
+case for a given language. In English it will be either ``one`` or ``other``.
```

Comment:
I'd add an explanation on when to use NUMBER($foo, {}). I.e., if you specify a particular number format in the placables, you should use the same number format in the plural rule, so that things like ordinals and trailing digits are handled correctly.